dz_dynamodb/sync_strategies/log_based.py

```diff
@@ -147,6 +156,10 @@ def sync(config, state, stream):
     table_name = stream['tap_stream_id']
 
     client = dynamodb.get_client(config)
+    # for open shards their will be a continous loop while getting records so to handle that max_iteration_for_open_shards is used
+    max_iteration_for_open_shards = 0
+    if config.get('real_time_data_view') :
```

Comment:
```suggestion
    if config.get('real_time_data_view') :
```
Add check for true as well